parsl/executors/taskvine/install-taskvine.sh

```diff
@@ -12,7 +12,7 @@ TARBALL="cctools-$CCTOOLS_VERSION-x86_64-ubuntu20.04.tar.gz"
 
 # If stderr is *not* a TTY, then disable progress bar and show HTTP response headers
 [[ ! -t 1 ]] && NO_VERBOSE="--no-verbose" SHOW_HEADERS="-S"
-wget "$NO_VERBOSE" "$SHOW_HEADERS" -O /tmp/cctools.tar.gz "https://github.com/cooperative-computing-lab/cctools/releases/download/release/$CCTOOLS_VERSION/$TARBALL"
+wget -c "$NO_VERBOSE" "$SHOW_HEADERS" -O /tmp/cctools.tar.gz "https://github.com/cooperative-computing-lab/cctools/releases/download/release/$CCTOOLS_VERSION/$TARBALL" || true
```

Comment:
what failure mode are you overriding here with || true?

Reply:
Well, egg on my face.  I was going to say: _In the "I use this locally" case, the `-c` parameter appears to return a non-zero if the file is already downloaded._  But going to prove that _now_, with a copied-from-my-terminal interaction, I'm no longer getting the error I thought I was working around:
```console
$ wget -c -O /tmp/cctools.tar.gz "https://github.com/cooperative-computing-lab/cctools/releases/download/release/7.7.2/cctools-7.7.2-x86_64-ubuntu20.04.tar.gz"
    ...
        The file is already fully retrieved; nothing to do.

$ echo $?
0
```

So ... nevermind.  I'll remove this hunk.